deploy/demo/contact-form/contact-form-ingress.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Hans Olav ønsker å fjerne ingress fra deploy og ha det sentralt. Snakk med han om det.

Reply:
@valosnah Vi hadde en kort prat om det i går, hvor vi først fjernet, og så lagt den tilbake igjen. Skal den likevel fjernes?

Reply:
Blir enklere med egen ingress her om den skal leve side om side med den eksisterende løsningen en stund

Reply:
Hva synes du om å bytte ut prefiksen `new` med noe annet, f.eks `v2`, `frontend`, eller noe sånt?

Reply:
lagde en møteinkalling for å avklare dette sammen med andre på mandag.